### PR TITLE
Support --version.

### LINKED ISSUE #1316: Support --version
Title says it all.

### PR DESCRIPTION
**Pull request by bangerth**
Fixes #1316.

### CODE REVIEW

**Comment by bangerth** [comment]
For the record, for me this prints as follows:
```
aspect-build> ./aspect --version
-----------------------------------------------------------------------------
-- This is ASPECT, the Advanced Solver for Problems in Earth's ConvecTion.
--     . version 1.5.0-pre
--     . running in DEBUG mode
--     . running with 1 MPI process
--     . using Trilinos
-----------------------------------------------------------------------------

Version information of underlying libraries:
   . deal.II:    8.5.0-pre
   . Trilinos:   12.2.1
   . p4est:      1.1.0

```

File: source/main.cc [hunk 1]
```diff
@@ -414,6 +413,37 @@ int main (int argc, char *argv[])
...
+          ((std::string (argv[1]) == "--version")
+           ||
+           (std::string (argv[1]) == "-v"))
+         )
+        {
+          // print usage info only on processor 0
+          if (Utilities::MPI::this_mpi_process(MPI_COMM_WORLD) == 0)
+            print_aspect_header(std::cout, true);
+
+          std::exit (0);
```
Comment (tjhei) [inline source/main.cc]: why ``std::exit`` and not ``return 0``?

File: source/main.cc [hunk 2]
```diff
@@ -414,6 +413,37 @@ int main (int argc, char *argv[])
...
+        }
+
+
+
+      // regular operation: read an input file and run with it
```
Comment (tjhei) [inline source/main.cc]: This comment confused me because I thought it explains this block. Maybe remove it?

**Comment by bangerth** [comment]
Like this, maybe?

**Comment by tjhei** [comment]
:+1:

**Pull request merged**